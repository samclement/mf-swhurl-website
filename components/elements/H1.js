import React from 'react'
import styled from 'styled-components'

const H1 = styled.h1`
  color: black;
`

export default ({ children }) => {
  return <H1>{children}</H1>
}
