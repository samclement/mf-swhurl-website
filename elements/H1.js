import React from 'react'
import styled from 'styled-components'

const H1 = styled.h1`
  color: deeppink;
`
H1.displayName = 'H1'

export default({children}) => {
  return (
    <H1>{children}</H1>
  )
}

