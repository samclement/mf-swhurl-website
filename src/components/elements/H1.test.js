/* eslint-env jest */

import React from 'react'
import { render } from '@testing-library/react'
import 'jest-styled-components'
import 'jest-dom/extend-expect'
import H1 from './H1.js'

describe('With Snapshot Testing', () => {
  it('H1 shows "mf.swhurl.com"', () => {
    const { asFragment } = render(<H1>mf.shwurl.com</H1>)
    expect(asFragment()).toMatchSnapshot()
  })
})
