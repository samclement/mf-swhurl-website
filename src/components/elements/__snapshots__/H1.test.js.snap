// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing H1 shows "mf.swhurl.com" 1`] = `
<DocumentFragment>
  .c0 {
  color: black;
}

<h1
    class="c0"
  >
    mf.shwurl.com
  </h1>
</DocumentFragment>
`;
